### DISCUSSION TITLE
共享的用户名和密码是什么

### DISCUSSION THREAD

**Discussion by zhangf1** [IceWhaleTech/CasaOS#1276] [Q&A]
![P30722-162232](https://github.com/IceWhaleTech/CasaOS/assets/103484452/72e48d4b-5346-4b85-8d71-62a320dcf096)
用电视端能搜索到设备，但是用户名和密码都不对

**Comment by Zhoucheng133** [comment]
这是文件共享么

好像文件共享的话没有用户名和密码的说法，我也试了一下用系统用户和密码的确不对，但是使用访客身份就直接可以完全访问了

你可以尝试一下用户名为guest，密码为空，至少通过smb的方式可行
